gapis/api/vulkan/externs.go

```diff
@@ -451,6 +451,9 @@ func (e externs) recordFenceReset(fence VkFence) {
 	}
 }
 
+func (e externs) recordWaitedFences(device VkDevice, fenceCount uint32, pFences VkFenceᶜᵖ) {
+}
```

Comment:
Why having this extern function if its body is empty?

Reply:
If an extern exists in the api file it MUST be defined in both go and cpp

Reply:
:+1: